jobs/Neatline_Job_DuplicateRecords.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/**
+ * @package     omeka
+ * @subpackage  neatline
+ * @copyright   2014 Rector and Board of Visitors, University of Virginia
+ * @license     http://www.apache.org/licenses/LICENSE-2.0.html
+ */
+
+
+class Neatline_Job_DuplicateRecords extends Omeka_Job_AbstractJob
```

Comment:
Hey @akstuhl and @jamiefolsom,

Great to see this feature, this will be super useful. Just to chime in really quickly - I think it would be fine, and maybe even preferable, for the records to be actually duplicated in the new exhibit, and not shared between them. Another use case for this that I've run into for this is wanting to do "version control" on entire exhibits, essentially, in a very low-fi way, sort of like saving off a copy of a Word document. Eg, before making a big set of content changes to an exhibit, I've often wanted to save off a copy of the exhibit, which I could in theory come back to if I decide I don't like the changes I'm about to make.

In terms of cloning big exhibits - this would cause problems if someone tried to duplicate a 1M record exhibit, but my guess is that like 99% of NL exhibits have fewer than about 1000 records, if not 100, so in practice I wouldn't worry too much about it.

Best,
-David